saw-core-coq/coq/_CoqProject

```diff
@@ -1,39 +1,15 @@
 -Q generated/CryptolToCoq   CryptolToCoq
 -Q handwritten/CryptolToCoq CryptolToCoq
--Q handwritten/Lift         Lift
```

Comment:
Now that these are removed, should the corresponding files under the `handwritten/Lift` directory also be removed? Similarly for `generated/S2N` and `handwritten/S2N`.

Reply:
Indeed, I should find a new home for them and remove them from this repository!

Reply:
OK. This might be worth opening a separate issue for, in case you're eager to land this.